queries/pokemon_query.go

```diff
@@ -0,0 +1,39 @@
+package queries
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/rromodev/academy-go-q32021/models"
+)
+
+func GetPokemonById(id int) models.Pokemon {
```

Comment:
It would be ok if this function could return an error too, there are a lot of places where this flow could potentially fail